.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I'll take it out. It was just handy to have a debugging environment out of the box, but it does not belong in the PR.